src/styles/components/icons/shapes/icon-tiny/styles.less

```diff
@@ -0,0 +1,78 @@
+& when (@icon-tiny-enabled) {
```

Comment:
We're missing a top level `& when (@icon-enabled)` condition here. Is it by intention?